docs/platforms/ios.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'iOS'
+---
+
+# iOS builds
+
+## Pre-requisites
+
+You must be building on a macOS machine, with Xcode installed. The Xcode installation must have an iOS SDK available, with all license agreements agreed to by the user. To check if an iOS SDK is available, open the Xcode settings panel, and check the Platforms tab. This will also ensure that license agreements have been acknowledged.
+
+Building iOS wheel also requires a working macOS Python configuration. See the notes on [macOS builds](./macos.md) for details about configuration of the macOS environment.
+
+## Specifying an iOS build
+
+iOS is effectively 2 platforms - physical devices, and simulators. While the API for these two platforms are identical, the ABI is not compatible, even when dealing with a device and simulator with the same CPU architecture. For this reason, the architecture specification for iOS builds includes *both* the CPU architecture *and* the ABI that is being targeted. There are three possible values for architecture on iOS; the values match those used by `sys.implementation._multiarch` when running on iOS:
+
+* `arm64-iphoneos` (for physical iOS devices);
+* `arm64-iphonesimulator` (for iOS simulators running on Apple Silicon macOS machines); and
+* `x64_64-iphonesimulator` (for iOS simulators running on Intel macOS machines).
+
+By default, cibuildwheel will build wheels for all three of these targets.
+
+If you need to specify different compilation flags or other properties on a per-ABI or per-CPU basis, you can use [configuration overrides](..#overrides) with a `select` clause that targets the specific ABI or architecture. For example, consider the following example:
+
+```
+[tool.cibuildwheel.ios]
+test-sources = ["tests"]
+test-requires = ["pytest"]
+
+[[tool.cibuildwheel.overrides]]
+select = "*-iphoneos"
+environment.PATH = "/path/to/special/device/details:..."
+
+[[tool.cibuildwheel.overrides]]
+select = "*-arm64-*"
+inherit.test-requires = "append"
+test-requires = ["arm64-testing-helper"]
+```
+
+This configuration would:
+
+ * Specify a `test-sources` and `test-requires` for all iOS targets;
+ * Add a `PATH` setting that will be used on physical iOS devices; and
+ * Add `arm64-testing-helper` to the test environment for all ARM64 devices (whether simulator or device).
```

Comment:
The only reason this doesn't match other ARM64 plaforms is because Windows and macOS don't  have a hyphen after `arm64`, and Linux calls it `aarch64`. But this is a bit too fragile to recommend. Suggest explicitly limiting the pattern to iOS.

Reply:
Resolved.